Doc/library/venv.rst

```diff
@@ -64,6 +64,8 @@ See :pep:`405` for more background on Python virtual environments.
 Creating virtual environments
 -----------------------------
 
```

Comment:
maybe it's worth adding an anchor for the link from the [cmdline page](https://docs.python.org/dev/library/cmdline.html) like for other modules
```suggestion
.. _venv-cli:
```
and update cmdline.rst:
```rst
* :ref:`venv <venv-cli>`
```